package/cfg/rke-cis-1.7-permissive/etcd.yaml

```diff
@@ -60,7 +60,7 @@ groups:
 
       - id: 1.1.20
         text: "Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Manual)"
-        audit: "find /etc/kubernetes/pki/ -name '*.crt' | xargs stat -c permissions=%a"
+        audit: "check_files_permissions.sh '/node/etc/kubernetes/ssl/!(*key).pem"
```

Comment:
The previous implementation got all files that end in `.crt` and checked their permissions. This version just gets all files that end with `.pem` but do not start with `key.pem` is this intended?